specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json

```diff
@@ -6954,6 +6996,12 @@
           "$ref": "#/definitions/PartnerNamespaceProperties",
           "description": "Properties of the partner namespace.",
           "x-ms-client-flatten": true
+        },
+        "systemData": {
```

Comment:
Does this property really exist in the API response ?

Reply:
Yes it does..